nautobot_secrets_providers/providers/azure.py

```diff
@@ -0,0 +1,70 @@
+"""Secrets Provider for Azure Key Vault."""
+
+import json
+
+try:
+  from azure.identity import DefaultAzureCredential
+  from azure.keyvault.secrets import SecretClient
+  azure_available = True
+except (ImportError, ModuleNotFoundError):
```

Comment:
This should be sufficient since `ModuleNotFoundError` is a subclass of `ImportError`
```suggestion
except ImportError:
```